app/auth/register/form.tsx

```diff
@@ -0,0 +1,30 @@
+'use client'
+import { FormEvent } from "react"
+
+export default function Form() {
+  
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+    const formData = new FormData(e.currentTarget);
+    const response = await fetch('/lib/auth/register', {
```

Comment:
@sealfury ah blast the compilation is failing because of this e.currentTarget. Any ideas?